src/vaadin-item-mixin.html

```diff
@@ -133,13 +152,15 @@
       }
     }
 
+    /** @private */
     _onKeydown(event) {
       if (/^( |SpaceBar|Enter)$/.test(event.key) && !event.defaultPrevented) {
         event.preventDefault();
         this._setActive(true);
       }
     }
 
+    /** @private */
```

Comment:
Please mark this listener as `@protected`, we [override it](https://github.com/vaadin/vaadin-tabs/blob/84cd9c794b113a6f429c717f0687bb381e98eb8d/src/vaadin-tab.html#L59-L62) in `vaadin-tab`.

Reply:
Done.